tests/puppeteer/DOCKER.md

```diff
@@ -16,6 +16,15 @@ docker-compose -f docker-compose.nightly.yml -f docker-compose.tests.yml exec -e
 docker-compose -f docker-compose.nightly.yml -f docker-compose.tests.yml up --build
 docker-compose -f docker-compose.nightly.yml -f docker-compose.tests.yml exec -e COMMAND="sanity-tests" tests bash /tmp/run-tests.sh
 ```
+
+### Functional tests
+
+```bash
+# Create Shop and running test
```

Comment:
Move this comment into the markdown part

Reply:
done thanks